staramr/subcommand/Search.py

```diff
@@ -313,6 +317,15 @@ def run(self, args):
         else:
             logger.info("No --pointfinder-organism specified. Will not search the PointFinder databases")
             pointfinder_database = None
+        
+        if(args.plasmidfinder_bacteria):
+            if args.plasmidfinder_bacteria not in PlasmidfinderBlastDatabase.get_available_bacteria():
+                raise CommandParseException("The only Plasmidfinder bacteria(s) currently supported are " + str(
```

Comment:
Instead of calling this `Plasmindfinder bacteria` could you change **bacteria** to something like **database type**? Could you make this change with the rest of your variable names as well?

Reply:
Yeah I can do that, I currently changed `plasmidfinder_bacteria` to `plasmidfinder_database_type` in `Search.py` file and changed `bacteria` to `database_type` in `PlasmidfinderBlastDatabase.py`